proto/gds/models/v1/models.proto

```diff
@@ -122,6 +128,12 @@ message GDSExtraData {
 
     // Log which records emails sent to the TRISA admins regarding this VASP
     repeated EmailLogEntry email_log = 6;
+
+    // Optional webhook for certificate delivery
+    string certificate_webhook = 7;
+
+    // Don't deliver certificates by email
+    bool no_email_delivery = 8;
```

Comment:
While it does make sense to put it on the extra data field; I would like to put it on the `VASP` record directly, since I think this is something we want to have the users be able to manage (e.g. they should be able to manage their delivery preferences). E.g. this is a TRISA-level field, not just something that is specific to the GDS.

Reply:
Alright that makes a bit more sense to me, especially if we're committing to doing the migration to include the new fields.